.travis.yml

```diff
@@ -1,16 +0,0 @@
-language: python
```

Comment:
let us remove travis.yml after the organisation settings are updated. Otherwise it may impact any PRS coming in between

Reply:
Already removed